qt/MainWindow.cc

```diff
@@ -859,6 +859,22 @@ void MainWindow::refreshStatusBar(TransferStats const& stats)
     }
 
     ui_.statsLabel->setText(str);
+    
+    if (prefs_.getBool(Prefs::INHIBIT_HIBERNATION))
```

Comment:
I don't know Qt client codebase at all, but I think you need to guard this with Windows specific macros.
This just won't compile for other supported systems